src/drive/styles/intentbutton.styl

```diff
@@ -0,0 +1,17 @@
+@require 'cozy-ui'
+
+// This ugly rules hide the default close button of the modal
```

Comment:
Is there really no other alternative ? Does the service really need to provide a close button ? What happen if for example the service is called within another container than an iframe ?

Reply:
What about removing the close button from service ?

Reply:
I want to keep the close button on the service to terminate correctly the intent.
+ removing the close button will probably have cross effect on other intents

Reply:
Dsl de revenir sur ceci mais je trouve ça vraiment pas terrible de mon coté.
Pourquoi ne pas ajouter une option au composant Modal ?
Nous avons une carte pour améliorer la Modal de cozy-ui je m'en occuperai à ce moment là.